client/angular/controllers/MainController.js

```diff
@@ -51,9 +51,19 @@
         };
 
         // Ordering function such that newer tweets precede older tweets
+        // Pinned tweets are always placed before unpinned tweets
         var chronologicalOrdering = function(tweetA, tweetB) {
-            return new Date(tweetB.created_at).getTime() - new Date(tweetA.created_at).getTime();
```

Comment:
did think of this, only problem is that if pinned tweets start to overflow onto the second/third column they would revert back to the standard chronological ordering which may seem inconsistent?